engine/src/main/java/org/terasology/rendering/nui/layers/ingame/metrics/DebugOverlay.java

```diff
@@ -142,6 +146,17 @@ public String get() {
                 }
             });
         }
+
+        UILabel debugLine5 = find("debugLine5", UILabel.class);
+        if (debugLine5 != null) {
+            debugLine5.bindText(new ReadOnlyBinding<String>() {
+                @Override
+                public String get() {
+                    return String.format("Executed final node task: %s", ((NodeTask) renderTaskSystem.getCursorTask()).getNode().getClass().getSimpleName());
```

Comment:
Actually, let me refine it:

"Last RenderNode processed: %s". 

And somehow this debugging mechanism will probably have to be restricted specifically only to render nodes (nodes writing to buffers), as I imagine that if one day we have additional nodes, i.e. cameras, buffers, acceleration structures, it wouldn't make sense to stop execution at one of those.

Also, we'll have to think about nodes that generate outputs that are not immediately visualizable or that are in an attachment of the FBO that is not visualized, I'm thinking the depth map generated by the shadow node and things like the normal buffer in the FBO sceneOpaque.

But for the time being of course this helps!